.github/workflows/test-deps.yml

```diff
@@ -7,14 +7,13 @@ on:
   workflow_dispatch:
 
 jobs:
-    test-nodevdeps:
+    test-latest-nodevdeps:
         runs-on: ${{ matrix.os }}
         strategy:
             fail-fast: false
             matrix:
-              # "macos-latest" removed for now as Docker action is slow
-              os: [ "ubuntu-latest" ]  # TODO: add "windows-latest"
-              python-version: ["3.8", "3.9", "3.10", "3.11"]
+              os: [ "ubuntu-latest", "windows-latest", "macos-latest" ]
+              python-version: ["3.10"]
```

Comment:
Why 3.10 in particular, not that it makes any difference?

The two logical ones are 3.8, our least common denominator, or 3.11, the latest and fastest.

Actually, black's linting takes into account the Python version, so we should probably use 3.8...?

Reply:
It was basically random, but you make a good point. I'll change to 3.8 now.